src/css/breadcrumbs.css

```diff
@@ -0,0 +1,16 @@
+.breadcrumbs__item--active .breadcrumbs__link {
+    color: var(--text-color) !important;
+}
+
+.breadcrumbs__item [aria-label="Home page"] {
+    line-height: 1.6;
+}
+
+.breadcrumbs__item:not(:last-child)::after  {
+    content: "\\" !important;
```

Comment:
based on the designs, shouldn't it be a slash `/` here?

Reply:
because if I use a single backslash ("\") it will be perceived that I'm trying to escape the character `"` when I really want to escape the character "\". So I need to use double back slashes

Reply:
oh sorry I think my msg wasn't clear, but I meant that in the designs it's a slash and not a backslash, just checking if this is intended

Reply:
oh, my bad. you are right. I'll fix it